design-proposals/live-migration-for-bridged-pod-network.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+This feature enables support of live-migrate for VMs with pod network in bridge mode.
+
+## Motivation
+Masquerade and slirp are not so performant compared to bridge mode.
+We want to use the less latency modes with the opportunity to live-migrate the virtual machines.
+
+## Goals
+Provide a live-migration feature for VMs running with pod network connected in bridged mode.
+
+## Non Goals
+Live migration with network reconfiguration is not seamless, sometimes it can take some time for applying a new settings.
+
+## Definition of Users
+Everyone who use bridge to bind pod network may want to live-migrate created VMs.
+
+## User Stories
+* As a user / admin, I want to have an opportunity for live-migration of a VM with bridged pod-network.
```

Comment:
I would actually be more interested in what apps / scenarios do **not** tolerate this kind of live migration.

Reply:
> This may affect some applications which are binding to the specific interface inside the VM.

Isn't it enough? Or should I specify cases with exact applications (eg. apache2 server configured to bind on pod IP instead of `0.0.0.0` and so on)?